_posts/fr/newsletters/2023-10-25-newsletter.md

```diff
@@ -0,0 +1,427 @@
+  concaténerait en un seul élément. Il fournit plusieurs descriptions des capacités que `OP_CAT` ajouterait à Script. Son
+  implémentation de référence proposée ne fait que 13 lignes de code (espaces blancs exclus).
+
+     La proposition a suscité une quantité modérée de discussions, principalement axées sur les limites de tapscript qui pourraient
+     affecter l'utilité et les coûts dans le pire des cas de l'activation de `OP_CAT` (et si certaines de ces limites devraient être
+     modifiées).
+
+## Sélection de Q&R du Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] est l'un des premiers endroits où les contributeurs
+d'Optech cherchent des réponses à leurs questions---ou lorsque nous avons quelques moments
+libres pour aider les utilisateurs curieux ou confus. Dans cette rubrique mensuelle, nous
+mettons en avant certaines des questions et réponses les plus appréciées, postées depuis
+notre dernière mise à jour.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+
+- [Comment fonctionne l'algorithme de sélection de pièces Branch and Bound ?]({{bse}}119919)
+  Murch résume son travail de recherche sur l'[algorithme Branch and Bound][branch
+  and bound paper] pour la [sélection de pièces][topic coin selection] qui "recherche l'ensemble d'entrées le moins gaspilleur
+  produisant une transaction sans changement".
+
+- [Pourquoi chaque transaction est-elle diffusée deux fois dans le réseau Bitcoin ?]({{bse}}119819)
+  Antoine Poinsot répond à un ancien message de la liste de diffusion de Satoshi qui indiquait "Chaque transaction doit être
+  diffusée deux fois". Poinsot précise que bien qu'à cette époque une transaction était diffusée deux fois (une fois lors de la
+  transmission de la transaction et une fois lors de la transmission du bloc), l'ajout ultérieur de [BIP152][] [compact block
+  relay][topic compact block relay] signifie que les données de transaction ne doivent être diffusées qu'une seule fois à un pair.
+
+- [Pourquoi les opcodes OP_MUL et OP_DIV sont-ils désactivés dans Bitcoin ?]({{bse}}119785)
+  Antoine Poinsot souligne que les opcodes `OP_MUL` et `OP_DIV` ont probablement été désactivés, en plus d'[autres opcodes][github
+  disable opcodes], en raison des bugs ["1 RETURN"]({{bse}}38037) et [OP_LSHIFT crash][CVE-2010-5137] découverts dans les semaines
+  précédentes.
+
+- [Pourquoi les hashSequence et hashPrevouts sont-ils calculés séparément ?]({{bse}}119832)
+  Pieter Wuille explique que, en divisant les données de hachage de transaction à signer en sorties précédentes et en séquences,
+  ces valeurs de hachage peuvent être utilisées une fois pour toute la transaction impliquant tous les types de sighashes.
+
+- [Pourquoi Miniscript ajoute-t-il une vérification de taille supplémentaire pour les comparaisons de préimages de hachage ?]
```

Comment:
This bullet is not getting a link. Im not 100% sure, but perhaps get the ({{bse}}119892) on the same line as the title